app.js

```diff
@@ -0,0 +1,29 @@
+//IMPORTACIONES
+const express = require("express")
+const app =express()
+require("dotenv").config()
+
+//MIDDLEWARES
+app.use(express.static("public"))
+app.set("views", __dirname + "/views")
+
+//RUTAS-- creacion de rutas
+
+app.get("/index.html", (req, res)=>{

```

Comment:
Te recomiendo teener una ruta raiz 
```suggestion
app.get("/", (req, res)=>{

```